config/config-artifact-bucket.yaml

```diff
@@ -1,33 +0,0 @@
-# Copyright 2019 The Tekton Authors
```

Comment:
config-artifact-pvc.yaml should go too

Reply:
Thanks @afrittoli removed config-artifact-pvc.yaml